projects/core/styles/mixins/mixins.less

```diff
@@ -129,3 +129,20 @@
     overflow: hidden;
     padding: 0;
 }
+
+// prevent scrolling to focused element on IOS
+.tui-prevent-ios-scroll() {
```

Comment:
Why we need prevent scroll?

Reply:
This is for fixed textfields like searchbar. IOS Safari tries to scroll to the textfield and the interface shakes